dev/PersonPicture/PersonPicture.cpp

```diff
@@ -141,64 +138,39 @@ void PersonPicture::UpdateIfReady()
     winrt::hstring initials = GetInitials();
     winrt::ImageSource imageSrc = GetImageSource();
 
-    if (auto initialsTextBlock = m_initialsTextBlock.get())
+    auto templateSettings = winrt::get_self<PersonPictureTemplateSettings>(TemplateSettings());
+    templateSettings->EffectiveInitials(initials);
+    if (imageSrc)
     {
-        if ((initials.empty() && !imageSrc) || IsGroup())
-        {
-            // Setting to the Person or Group asset, ensure the correct FontFamily.
-            initialsTextBlock.FontFamily(winrt::FontFamily(L"Segoe MDL2 Assets"));
-
-            // If true, change the symbol block to show the Group asset. Otherwise show the Person asset.
-            winrt::hstring content = (IsGroup()) ? L"\uE716" : L"\uE77B";
-            initialsTextBlock.Text(content);
-        }
-        else
-        {
-            // Change the FontFamily of text block to original setting
-            initialsTextBlock.FontFamily(m_personTextFontFamily.get());
-
-            // Assign the initials of contact to text block 
-            winrt::hstring content = (!ProfilePicture()) ? initials : L"";
-            initialsTextBlock.Text(content);
-        }
+        winrt::ImageBrush imageBrush;
```

Comment:
I suggest we check if templateSettings->EffectiveImageBrush.ImageSource == imageSrc. Otherwise we will trig a  lot of propertychanged event since UpdateIfReady is called everywhere

Reply:
I'll fix to avoid recreating the ImageBrush if we don't need to, but I'll leave the image source check to the implementation of ImageBrush.